src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
you forget to change the [selected] icon in menus + it's not centered 
![image](https://user-images.githubusercontent.com/31045223/221853858-389a03c7-8114-4a63-a21a-68f608f364cd.png)